python/batch-predictor/env310.yaml

```diff
@@ -0,0 +1,8 @@
+name: merlin-model
```

Comment:
Can we move all those environment yaml files (env37, env38, etc) to its own directory?